terra-workspace-manager-jmeter/src/main/java/automation/jmeter/CreateWorkspaceSimulation.java

```diff
@@ -0,0 +1,307 @@
+                "org.apache.jmeter.visualizers.backend.influxdb.HttpMetricsSender", "=");
+        Argument influxdbUrl = new Argument("influxdbUrl", "${__env(INFLUXDB_WSM_URL)}", "=");
+        Argument application = new Argument("application", "Terra Workspace Manager", "=");
+        Argument measurement = new Argument("measurement", "${__env(INFLUXDB_WSM_MEASUREMENT)}", "=");
+        Argument summaryOnly = new Argument("summaryOnly", "false", "=");
+        Argument samplersRegex = new Argument("samplersRegex", ".*", "=");
+        Argument percentiles = new Argument("percentiles", "90;95;99", "=");
+        Argument testTitle = new Argument("testTitle", "Test name", "=");
+        Argument eventTags = new Argument("eventTags", "", "=");
+
+        props.addItem(influxdbMetricsSender);
+        props.addItem(influxdbUrl);
+        props.addItem(application);
+        props.addItem(measurement);
+        props.addItem(summaryOnly);
+        props.addItem(samplersRegex);
+        props.addItem(percentiles);
+        props.addItem(testTitle);
+        props.addItem(eventTags);
+
+        Arguments arguments = new Arguments();
+        arguments.setName("arguments");
+        arguments.setEnabled(true);
+        arguments.setProperty(TestElement.TEST_CLASS, Arguments.class.getName());
+        arguments.setProperty(TestElement.GUI_CLASS, ArgumentsPanel.class.getName());
+        arguments.setProperty(props);
+
+        backendListener.setArguments(arguments);
+    }
+
+    private static HashTree configureTestPlan() {
+        testPlanTree = new HashTree();
+        testPlanTree.add(testPlan);
+
+        HashTree setUpThreadGroupTree = new HashTree();
+        setUpThreadGroupTree.add(threadGroup);
+        setUpThreadGroupTree.add(setupThreadGroup);
+
+        HashTree parametersTree = new HashTree();
+        parametersTree.add(headerManager);
+        parametersTree.add(userParameters);
+        parametersTree.add(backendListener);
+
+        HashTree httpSamplerTree = new HashTree();
+        httpSamplerTree.add(httpSamplerProxy);
+        httpSamplerTree.add(httpSamplerProxy, parametersTree);
+
+        setUpThreadGroupTree.add(threadGroup, httpSamplerTree);
+
+        HashTree jsr223PreProcessorTree = new HashTree();
+        jsr223PreProcessorTree.add(jsr223PreProcessor);
+        setUpThreadGroupTree.add(setupThreadGroup, jsr223PreProcessorTree);
+
+        testPlanTree.add(testPlan, setUpThreadGroupTree);
+
+        return testPlanTree;
+    }
+
+    public static void main(String[] argv) throws Exception {
+
+        // for capturing HTTP traffic
+        // System.setProperty("http.proxyHost", "127.0.0.1");
+        // System.setProperty("http.proxyPort", "8888");
+        // for capturing HTTPS traffic
+        // System.setProperty("https.proxyHost", "127.0.0.1");
+        // System.setProperty("https.proxyPort", "8888");
+
+        File jmeterHome = new File("/apache-jmeter-5.3");
```

Comment:
The test(s) will need to be updated when the jmeter version changes in the gradle file, correct? Any way we can make it more seamless (maybe fetch via a gradle task or something)?

Reply:
The Jmeter location will be replaced by an envvar (JMETER_HOME), this applies to all environments. Added a TODO for this.